care/emr/api/viewsets/inventory/inventory_item.py

```
@@ -32,7 +35,12 @@ class InventoryItemViewSet(EMRRetrieveMixin, EMRListMixin, EMRBaseViewSet):
     pydantic_retrieve_model = InventoryItemRetrieveSpec
     filterset_class = InventoryItemFilters
     filter_backends = [filters.DjangoFilterBackend, OrderingFilter]
-    ordering_fields = ["created_date", "modified_date", "net_content"]
+    ordering_fields = [
+        "created_date",
+        "modified_date",
+        "net_content",
+        "product__expiration_date",
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)